client/src/components/App.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props),
+    this.state = {}
+  }
+  render() {
+    return(
+      <div>
+        <h1>Hello</h1>
+      </div>
+    )
+  }
+}
+
+
+export default App;
```

Comment:
Seems like github wants you to add a new empty line at the end of each file for some reason.
But also you already got a little React set up! good for you! I haven't yet.